test/amm/minterAmmRemoveExpired.js

```diff
@@ -0,0 +1,337 @@
+const { time, expectEvent, BN } = require("@openzeppelin/test-helpers")
+const Market = artifacts.require("Market")
+const Proxy = artifacts.require("Proxy")
+const SimpleToken = artifacts.require("SimpleToken")
+const MinterAmm = artifacts.require("MinterAmm")
+const MarketsRegistry = artifacts.require("MarketsRegistry")
+const MockPriceOracle = artifacts.require("MockPriceOracle")
+const { MarketStyle, getPriceRatio, checkBalances } = require("../util")
+
+const ONE_DAY = 86400
+const THIRTY_DAYS = 30 * ONE_DAY
+const THIRTY_TWO_DAYS = 32 * ONE_DAY
+const BTC_ORACLE_PRICE = 14_000 * 10 ** 8 // BTC oracle answer has 8 decimals places, same as BTC
+
+const STATE_OPEN = 0
+const STATE_EXPIRED = 1
+
+contract("Minter AMM Remove expired markets", (accounts) => {
```

Comment:
Thanks for these tests; they cover  most of the edge cases I'm worried about.

Before merging, can you add additional checks in all 3 of the tests that verify the ending state of the `openMarkets` `address[]` is what we expect? So for instance for the first test you wrote, we should assert that `assert.equal(0, (await deployedAmm.getMarkets()).length)`, and for the second test we should see that:

we start with the ordering:
`[marketAddress1, marketAddress2, marketAddress3, marketAddress4, marketAddress5]` and then we end with `[marketAddress1, marketAddress2, marketAddress4]`